examples/mongodb/1.8/Dockerfile

```diff
@@ -2,13 +2,19 @@ FROM mongo:5.0
 
```

Comment:
Debezium 1.8 will come with Mongo DB 5. Did you verify the commands work with Mongo DB 5?

Reply:
yep! That's the problem I explain in my other comment below. You will always need a second (empty) Mongo DB instance to initialize the replica set.
In the Debezium UI I do this here:
https://github.com/debezium/debezium-ui/blob/main/docker-compose.yml#L55-L61

Reply:
is this means we need to initialize the replica set from shell with separate command?